src/ApplePay/Decoding/PKCS7SignatureValidator.php

```diff
@@ -129,7 +129,22 @@ private function extractCertificates($signature)
 
         $certificates = $this->openSslService->getCertificatesFromPkcs7($pkcs7TemporaryFile->getPath());
 
-        return explode("\n\n", $certificates); // the response contains 2 certificates; separate them into an array
+        // The response contains 2 certificates. Let's extract them by their ends.
+        // It's good idea to ensure that certificates are at proper format. Some environments
+        // may ended up with additional "\n" characters.
+        //
+        $certificates = explode(
+            "-----END CERTIFICATE-----\n",
```

Comment:
I think a valid certificate does not always end in "END CERTIFICATE" but can also end in "END PKCS7"